src/GradeScale.ts

```diff
@@ -10,7 +10,7 @@ export default interface GradeScale {
   displayName: string
   name: GradeScalesTypes
   offset: number
-  allowableConversionType: GradeScalesTypes[]
+  conversionGroup: string
```

Comment:
I guess there is a better way to define a finite set of options

Reply:
Yes, some sort of enum might be more robust than a string.

Reply:
Do you want to change this in this PR or leave it for later?